app/konnect/upgrade-faq.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Frequently asked questions - Control Plane Upgrades
+---
+
+
+
+### Are {{site.konnect_short_name}} control plane and associated database migrations or upgrades done by Kong Inc
+
+The {{site.base_gateway}} control plane and its dependencies are fully managed by Konnect. As new versions of {{site.base_gateway}} are released, Konnect supports them as long as they are under our active support schedule.
+
+
+### Will {{site.konnect_short_name}} control plane upgrades always show incompatible message on runtime manager page if the data planes are not the same version as the  {{site.konnect_short_name}} control plane?
+
+An old configuration may still be 100% compatible with older data planes and therefore not show any error messages in the {{site.konnect_short_name}} UI. If there are compatibility issues detected when pushing the payload down to the data plane then
```

Comment:
```suggestion
An old configuration may still be 100% compatible with older data planes and therefore not show any error messages in the {{site.konnect_short_name}} UI. If there are compatibility issues detected when pushing the payload down to the data plane then this will be reflected in the UI.
```